docs/faq/users/prefer-gpu-to-cpu.rst

```diff
@@ -1,26 +1,32 @@
 Example: Run a Job on a CPU if a GPU is not Available
 =====================================================
 
+Some jobs may be able to take advantage of GPU if one is available, but
+can run on a CPU if there are no slots with GPUs immediately matchable.
+The following submit file will match a job in this case.
+
 .. code-block:: condor-submit
 
-    ## Magic section: don't change anything in this section unless you
-    ## understand why it works.
-    rank                        = RequestGPUs
-    # If your job requires more than one GPU, you may change both
-    # of the '1's in the next line to some other number.
-    request_GPUs                = countMatches(RequireGPUs, AvailableGPUs) >= 1 ? 1 : 0
+    ## Execution section. You will need to change these for your job
```

Comment:
Add link to the current page, so when user copy/pastes they can come back to this document?

Reply:
This is fantastic, thanks!